run/deployment-previews/README.md

```diff
@@ -1,25 +1,522 @@
+
+1. On your GitHub repo page, click **Add file** > **Create new file**
+    1. Name the new file `cloudbuild-preview.yaml`
+    1. Copy the code into your new file: [cloudbuild-preview.yaml](/run/deployment-previews/cloudbuild-configurations/cloudbuild-preview.yaml)
+2. Commit the change to the main branch of your repository.
+
+
+## Creating the secondary trigger
+
+Now that all the groundwork is in place, create the new trigger.
+
+1. Create a new Cloud Build trigger ([full instructions](https://cloud.google.com/build/docs/automating-builds/create-manage-triggers#console)):
+    1. In the Google Cloud console, go to the **Cloud Build triggers** page.
+
+       <a href="https://console.cloud.google.com/cloud-build/triggers"
+           target="console"
+           track-type="tutorial"
+           track-name="consoleLink"
+           track-metadata-position="body"
+           class="button button-primary">Go to the Cloud Build triggers page</a>
+
+    1. Click **Create Trigger**.
+    1. Enter the following details:
+      *   Name: `preview-deploy`
+      *   Event: Pull Request
+      *   Source Repository: "<var>GITHUB_USER_NAME</var>/helloworld-python"
+      *   Base branch: `^main$`
+      *   Comment control: Required except for owners and collaborators
+         -  As the owner of the repo, previews will be automatically built on pull requests you create.
+         -  If you want to allow anyone to preview their changes, read more about the [security implications](https://cloud.google.com/build/docs/cloud-build-service-account) of selecting "Not required".
+      *   Configuration: Cloud Build Configuration file
+      *   Cloud Build configuration file location: `cloudbuild-preview.yaml`
+    1. Click **Create**.
+
+
+## Checking Success
+
+Since this new trigger is fired when a new pull request is created, you will
+need to create a new pull request in order to test it.
+
+1. Go to your repository, and make a visual change to `app.py` in a
+[new branch](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository).
+    1. Go to `app.py`, and click the pencil icon (<span class="material-icons">edit</span>).
+    <figure id="github-edit-file">
+      <img
+        src="images/deployment-previews-github-edit-file-2.png"
+        alt="Screenshot of the GitHub interface for editing files on the website." />
+    </figure>
+
+    1. Make a change; for example, change "Hello" to "Greetings".
+    1. Select **Create a new branch for this commit and start a pull request**, then click **Propose change**.
+1. Create a [new pull request](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request) with this branch.
+
+  If the trigger is configured correctly, a new check is displayed soon after
+you create the pull request:
+
+    <figure id="github-pending-check">
```

Comment:
Figure not rendering. You may need to check the indentation or not have a newline between the figure and text